libbpfgo.go

```diff
@@ -440,6 +440,33 @@ func (b *BPFMap) GetMaxEntries() uint32 {
 	return uint32(maxEntries)
 }
 
+func (b *BPFMap) GetFd() int {
+	return int(b.fd)
+}
+
+func (b *BPFMap) GetName() string {
+	return b.name
+}
+
+func (b *BPFMap) GetPinPath() string {
+	cs := C.CString(b.name)
+	bpfMap := C.bpf_object__find_map_by_name(b.module.obj, cs)
+	pinPathGo := C.GoString(C.bpf_map__get_pin_path(bpfMap))
+	C.free(unsafe.Pointer(cs))
+	return pinPathGo
+}
+
```

Comment:
Could you add a selftest for GetPinPath() and IsPinned() ?
Something very simple, making sure they work and don't regress ?

> Note: You can clone a simple examples from selftest/ directory.

Reply:
@rafaeldtinoco, done. Thanks a million.

Reply:
I like it. I'll let sit here for awhile (to see if others have any comment). Will run the selftest together with others and if all good will commit. Thank U for the contribution.